src/dlgTriggerEditor.cpp

```diff
@@ -491,14 +491,14 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
     }
 
     mpAction_searchOptions = new QAction(tr("Search Options"), this);
-    mpAction_searchOptions->setObjectName(QLatin1String("mpAction_searchOptions"));
+    mpAction_searchOptions->setObjectName(QStringLiteral("mpAction_searchOptions"));
 
     QMenu* pMenu_searchOptions = new QMenu(tr("Search Options"), this);
-    pMenu_searchOptions->setObjectName(QLatin1String("pMenu_searchOptions"));
+    pMenu_searchOptions->setObjectName(QStringLiteral("pMenu_searchOptions"));
     pMenu_searchOptions->setToolTipsVisible(true);
 
     mpAction_searchCaseSensitive = new QAction(tr("Case sensitive"), this);
-    mpAction_searchCaseSensitive->setObjectName(QLatin1String("mpAction_searchCaseSensitive"));
+    mpAction_searchCaseSensitive->setObjectName(QStringLiteral("mpAction_searchCaseSensitive"));
```

Comment:
Ah, good catch - BTW how does `QLatin11String` compare to `QLatin1Literal` I have seen a few uses of the latter used in our code-base and I suspect that is the QtCreator IDE _helpfully_ auto-completing it when the other one is really wanted... 😮